project-metrics/metrics_service/apis/codecov_test.py

```diff
@@ -0,0 +1,46 @@
+"""Tests for Codecov API interface."""
+
+from agithub import base as agithub_base
+import unittest
+from unittest import mock
+
+import env
+from apis import codecov
+
+
+class TestCodecovApi(unittest.TestCase):
+
+  def setUp(self):
+    super(TestCodecovApi, self).setUp()
+
+    mock.patch.object(
+        env, 'get', side_effect={
+            'GITHUB_REPO': '__repo__',
+        }.get).start()
+    self.mock_request = mock.patch.object(
+        agithub_base.Client, 'request', autospec=True).start()
```

Comment:
You know what I think about mocking an implementation detail of a 3rd party library but alright ;)

Reply:
Yep yep :)